go-controller/pkg/ovn/default_network_controller.go

```diff
@@ -846,6 +855,7 @@ func (h *defaultNetworkControllerEventHandler) AddResource(obj interface{}, from
 // Given an old and a new object; The inRetryCache boolean argument is to indicate if the given resource
 // is in the retryCache or not.
 func (h *defaultNetworkControllerEventHandler) UpdateResource(oldObj, newObj interface{}, inRetryCache bool) error {
+	var aggregatedErrors []error
```

Comment:
please define this closer to where its actually being used? why is this here?

Reply:
Moved it under NodeType case, Thanks!